### PR TITLE
Index Website Readability Update

### PR DESCRIPTION
**Pull request by tytarman**
# **Pull Request Template**

## **Type of Change**

*Select a category that relates to the content of your pull request (choose one).*

- [ ] Bug fix
- [ ] New feature
- [ ] Documentation
- [ ] Code Refactoring
- [x] Update of code styling

&nbsp;

## **Does this request include a breaking change?**

*Is there any potential for this change to crash Index UI or make it unusable?*

- [ ] Yes
- [x] No [this should just be a styling update]

&nbsp;

## **Description**

*Provide a brief summary of the changes you made and their purpose as they relate to the Index UI.*

*If you are resolving an issue, please link the corresponding issue number.*

&nbsp;

### **Summary of Changes:**

- Provided an update to the styling of the min/max indicators on the price charts.
- Updated the graph animation speed to be more responsive.
- Fixed a text overflow issue on the Vote Proposal table.

### **Fixes:** Issue #267

&nbsp;

## **Test Data or Screenshots**

*Include proof that your changes function properly and resolve the linked issue.*

## **Before**

min:max (before)

&nbsp;

text overflow (before)

&nbsp;

## **After**

min:max desktop (after)

&nbsp;

min:max iphone (after)

&nbsp;

text overflow (after)

&nbsp;

## **Checklist**

*Please review and confirm each of the following before creating your pull request.*

- [x] I have reviewed the [Contribution Guidelines](https://github.com/SetProtocol/index-ui/blob/master/CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [x] I have checked my code and corrected any errors.
- [x] I have commented my code or used an effective naming scheme.
- [x] I have updated my repository to match the master at `SetProtocol/index-ui`.
- [x] I have included test data or screenshots that prove my fix is effective or that my feature works.

&nbsp;

*If all of the above are **true**, then you may mark your pull request as **ready for review**.*

### CODE REVIEW

File: src/components/SimplePriceChart/SimplePriceChart.tsx [hunk 1]
```diff
@@ -127,16 +132,23 @@ const MarketDataChart: React.FC<SimplePriceChartProps> = ({
...
             stroke={theme.colors.grey[500]}
-            tickFormatter={(n) => '$' + formatFloats(n)}
+            tickFormatter={tickFormatter}
             axisLine={false}
             tickLine={false}
             mirror={true}
-            tick={{ fontFamily: 'Roboto Mono' }}
-            ticks={[minimumYAxisLabel, maxY + 5]}
-            domain={[minY - 10, maxY + 10]}
+            tick={{ fontFamily: '' }}
```
Comment (0xModene) [inline src/components/SimplePriceChart/SimplePriceChart.tsx:144]: I'm going to say either turn this back to Roboto Mono, or remove the `tick=` altogether. No point in keeping if its empty

File: src/views/Vote/components/ProposalTable.tsx [hunk 2]
```diff
@@ -140,6 +140,7 @@ const StyledColumn = styled.div<StyledColumnProps>`
   padding: 5px 10px 5px 30px;
   align-items: center;
   display: flex;
+  overflow: scroll;
```
Comment (0xModene) [inline src/views/Vote/components/ProposalTable.tsx:143]: Nice catch!

**Review by 0xModene — CHANGES_REQUESTED** [review]
There's a minor change I'd like to see, but mostly looks good

File: src/views/Vote/components/ProposalTable.tsx (hunk 2 above)
Comment (0xModene) [inline src/views/Vote/components/ProposalTable.tsx:143]: Nice catch!

File: src/components/SimplePriceChart/SimplePriceChart.tsx (hunk 1 above)
Comment (tytarman) [reply src/components/SimplePriceChart/SimplePriceChart.tsx]: Fair point. Removing the whole line seems like the best approach.

**Review by 0xModene — APPROVED** [review]
lgtm

**Pull request merged**